variables.tf

```diff
@@ -130,6 +130,12 @@ variable "secrets" {
   description = "Map containing secrets to expose to the docker container"
 }
 
+variable "service_launch_type" {
+  type        = string
+  default     = "FARGATE"
+  description = "The service launch type: either FARGATE or EC2"
+}
```

Comment:
Can you add a validation check on this? So we can ensure it's always one of these 2 values?

See: https://www.terraform.io/docs/language/values/variables.html#custom-validation-rules

Something like this:
``` 
 validation {
    condition     = contains(["FARGATE", "EC2", "eur", "nsa"], var. service_launch_type)
    error_message = "Allowed values for service_launch_type are \"FARGATE\",  or \"EC2\"."
  }